getting-started/windows-calico/quickstart.md

```diff
@@ -29,8 +29,23 @@ Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Window
 - Remote access to the Windows node via Remote Desktop Protocol (RDP) or Windows Remote Management (WinRM)
 - Be able to run a command as Administrator using powershell.
 - Additionally, for EKS:
-    - The VPC controllers must be installed be installed to run Windows pods.
-    - The Windows instance role must have access to `secrets` in the calico-system namespace or kube-system namespace if you are using a non operator-managed Calico installation.
+  - The VPC controllers must be installed be installed to run Windows pods.
+  - The Windows instance role must have permissions to get `namespaces` and get `secrets` in the calico-system namespace (or kube-system namespace if you are using a non operator-managed {{site.prodname}} installation.)
+    - For non-production usage, run these commands below to install the permissions needed to install {{site.prodnameWindows}}.
```

Comment:
Why `For non-production usage`? We've updated installation script to enable WinDSR by default so we should claim EKS support production ready.

Reply:
I was just thinking that maybe there is a better way of doing this. I'll reword that.
I also need to replace the usage of the `view` role with something just to calico or kube-system get/list namespaces.